test/unit/plugins/guests/linux/cap/shell_expand_guest_path_test.rb

```diff
@@ -25,6 +25,14 @@
       cap.shell_expand_guest_path(machine, path)
     end
 
+    it "expands a path with tilde" do
+      path = "~/folder"
+      allow(machine.communicate).to receive(:execute).
```

Comment:
```suggestion
      expect(machine.communicate).to receive(:execute).
```

Reply:
If this is RSpec and this https://relishapp.com/rspec/rspec-mocks/v/3-9/docs/basics/expecting-messages explains it right, a failed expect will fail the test...which should be the preferred action, shouldn't it?